studio.py

```diff
@@ -30,14 +232,26 @@
     help="Proxy Service to use.",
 )
 @click.option("--record/--replay", default=False)
-@click.option("--certutil", help="Path to certutil.")
-@click.option("--url", default="about:blank", help="Site to load.")
-@click.argument("path")
+@click.option(
+    "--certutil", help="Path to certutil. Note: Only when recording and Only on Android"
+)
+@click.option(
+    "--sites",
+    help="JSON file containing the websites information we want ro record. Note: Only when recording",
+)
+@click.argument(
+    "--path", default="Recordings", help="Path where to store/are stored the recordings"
+)
+@click.option(
+    "--url", default="about:blank", help="Site to load. Note: Only when recording."
```

Comment:
Why is this only when recording?

Reply:
oups... *replaying

when we replay we need a file and URL. When we record we read the provided json file.